README.md

```diff
@@ -440,6 +493,52 @@ We're providing other DV360 operators, to be used in your DAGs, so that you're a
 </table>
 
 
+# Structured Data Files
+
+Below we will explain how to set up a workflow which will import your [DV360 Structured Data Files (SDFs)](https://developers.google.com/bid-manager/guides/structured-data-file/format) to BigQuery.
+
+### Create a new Airflow Connection or update an existing one
+
+If you haven’t created an Airflow Connection for GMP APIs follow [Create the Airflow Connection to GMP reporting](#create-the-airflow-connection-to-gmp-reporting) step to create one. Make sure that in the last step, the following scopes are added: 
+
+https://www.googleapis.com/auth/doubleclickbidmanager,
+https://www.googleapis.com/auth/devstorage.full_control,
+https://www.googleapis.com/auth/bigquery
+
+### Create an SDF advertisers report
+
+This report will contain all active advertiser IDs with their Partner IDs which will be used to retrieve SDFs via API. To create a new SDF advertisers report, in Airflow, please manually run:
+
+dv360_create_sdf_advertisers_report_dag
+
+The above DAG will create a scheduled DV360 report which will run daily. After it’s successfully completed, you should see that “dv360_sdf_advertisers_report_id” Airflow variable has updated with a newly created report ID. 
+
+Note: These scheduled reports expire on the 1st of January 2029.
+
+### Run the SDF advertisers report
+
+Once you’ve created the SDF advertisers report, please manually run the following DAG:
+
+dv360_run_sdf_advertisers_report_dag
+
+After it’s completed, manually run:
+
+dv360_get_sdf_advertisers_from_report_dag
+
+After it’s completed, you should be able to verify that  “dv360_sdf_advertisers” Airflow variable now contains relevant partner and advertiser IDs which will be used to retrieve SDFs. The above DAG will be automatically configured to run daily.
+
+### Upload SDFs to BigQuery
+
+Please manually run the following DAG:
+
+dv360_sdf_uploader_to_bq_dag
+
+To upload SDFs to BigQuery. The process will use the dictionary stored in the “dv360_sdf_advertisers” Airflow variable to make API requests and store responses in your BigQuery dataset. 
+
+Once the DAG has completed successfully, you will find new tables in your BigQuery dataset. Tables will correspond to SDF types you’ve configured to retrieve in the “sdf_file_types” Airflow variable (e.g. if you’ve configured “LINE_ITEMS”, you should see a table called “SDFLineItem”). The above DAG will be automatically configured to run daily.
+
+To sum up, we’ve scheduled two DAGs which run independently from each other. The first DAG downloads a report and updates an Airflow variable with your partner and advertiser IDs. The second DAG fetches Structured Data Files using partner and advertiser IDs from the Airflow variable and uploads them to BigQuery.
```

Comment:
Nit, since the dag is now scheduled to run daily can you add this step to the summation?